### DISCUSSION TITLE
win cannot connect to Linux in LAN, but  can connect to Windows.

### DISCUSSION THREAD

**Discussion by WindDaoist** [rustdesk/rustdesk#9854] [Q&A]
![image](https://github.com/user-attachments/assets/b8c4daac-d343-4183-b996-ee46f61c1724)

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/discussions/763, go to [discord](https://discord.gg/nDceKgxnkV) please, please search there first, many guys have asked similar question because you are not the most special though you always think you are. Usually nobody will help you with build or self-hosting here.
